spec/requests/providers/means/cash_outgoings_controller_spec.rb

```diff
@@ -0,0 +1,130 @@
+require "rails_helper"
+
+RSpec.describe Providers::Means::CashOutgoingsController, type: :request do
```

Comment:
```suggestion
RSpec.describe "Providers::Means::CashOutgoingsController", type: :request do
```

```suggestion
RSpec.describe "provider outgoings. request", type: :request do
```

Reply:
I have updated the file and renamed it to be consistent with the majority of other specs, however we do have instances where we use the model name in the spec and some where we have specs with 'controller' in the name. e.g. `full_employment_details_controller_spec.rb`. This appears to be due to govuk rubocop rules:
 RSpec/FilePath: Spec path should end with providers/means/cash_outgoings_controller*_spec.rb. (https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/FilePath)